shimmer/docker-compose.yml

```diff
@@ -0,0 +1,363 @@
+#
+# node-docker-setup __VERSION__
+# https://github.com/iotaledger/node-docker-setup
+#
+
+version: '3'
+services:
+
+##################################################################
+#  HORNET - Stardust                                             #
+##################################################################
+
+  hornet:
+    container_name: hornet
+    image: iotaledger/hornet:2.0
```

Comment:
should we keep the `-rc` labels here? Or do we want to make it rc with a version bump of the setup?

Reply:
I would say the latter. Let's keep it for now because the current rc version on docker hub is slightly older than the current release.